gengetopt.yaml

```diff
@@ -0,0 +1,142 @@
+package:
+  name: gengetopt
+  version: 2.23
+  epoch: 0
+  description: "A tool to write command line option parsing code for C programs"
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - libtool
+      - texinfo
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.gnu.org/gnu/gengetopt/gengetopt-${{package.version}}.tar.xz
+      expected-sha256: b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-static
+    pipeline:
+      - uses: split/static
+    description: gengetopt static
+
+  - name: ${{package.name}}-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    description: gengetopt dev
+
+  - name: ${{package.name}}-doc
```

Comment:
It seems main package still has a `doc` folder: https://apk.chaindag.dev/https/apk.cgr.dev/wolfi-presubmit/56d7982b85b02eca28b0964836ecc7ae8e7172a2/aarch64/gengetopt-2.23-r0.apk@etag:4306069eadac588dde8a0653d06a51d4/usr/share/

Reply:
@Dentrax Should that be manually removed?

Reply:
We should `mv` that doc dir to -doc subpkg i guess

Reply:
Thanks @Dentrax , docs moved